client/Index.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import axios from 'axios';
+import MainDescription from './components/MainDescription.jsx';
+class Index extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { description: null };
+  }
+
+  componentDidMount() {
+    axios.get('/api/description/0').then(({ data }) => {
+      this.setState({ description: data });
+    });
+  }
```

Comment:
@cmonez don't forget to handle your errors in the catch block 😷

Reply:
Oops. Forgot about that, i'll add that.